### PR TITLE
Fix for smart pointer bug

### PR DESCRIPTION
**Pull request by rdbaris**
Resolving an issue found in #11893 that came up when adding smart pointers

### CODE REVIEW

File: src/openrct2/CmdlineSprite.cpp [hunk 1]
```diff
@@ -90,17 +90,16 @@ static bool sprite_file_open(const utf8* path)
...
+        std::unique_ptr<rct_g1_element_32bit[]> openElements = std::make_unique<rct_g1_element_32bit[]>(
+            spriteFileHeader.num_entries);
         if (openElements == nullptr)
         {
             fclose(file);
             return false;
         }
 
-        if (fread(openElements, openEntryTableSize, 1, file) != 1)
+        if (fread(openElements.get(), spriteFileHeader.num_entries, 1, file) != 1)
```
Comment (ZehMatt) [inline src/openrct2/CmdlineSprite.cpp:101]: This is now incorrect, fread expects the total amount of bytes as the parameter, this passes only the amount of entries.

Comment (rdbaris) [reply src/openrct2/CmdlineSprite.cpp]: Sorry about that! This should fix it

**Comment by ZehMatt** [comment]
Looks about right now, this branch needs to rebased and it should be good to go.

**Comment by tupaschoal** [comment]
This page explains about sync and rebase @rdbaris 
https://github.com/OpenRCT2/OpenRCT2/wiki/Rebase-and-Sync-fork-with-OpenRCT2

**Pull request merged**